skins/universal/breath/Breath_Dark.skn

```diff
@@ -1,4 +1,4 @@
-<skin path="/home/vitaly/.local/share/psi+/skins/universal/breath" author="VitoZz" name="Breath_Dark" version="1.0">
```

Comment:
for the future, no need to delete this path, it will reappear with each new skin release, it is always added by the plugin when saving the skin